### DISCUSSION TITLE
cut out segments from in between ?

### DISCUSSION THREAD

**Discussion by SyedMuhamadYasir** [mifi/lossless-cut#1561] [Q&A] [answered]
### I have a lot of issues to go through, so in order to make it easier for me to help you, I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

is it possible to cut out a video segment from in between a video ?

**Comment by mifi** [comment]
I don’t know what that means

**Comment by SyedMuhamadYasir** [comment]
> I don’t know what that means

assume there's a 10 sec video - in which from timeframe 00:06 (sixth second) till timeframe 00:009 (ninth second) is the undesirable part of the video

and i want to edit it out of the original 10 second video ( so i will be joining the remaining part of the video from 00:05 and the final second of the video 00:10

basically , i want to edit out the part starting from 00:06 till 00:09, from the original 10 second video

how can i do that with your tool ?

**Reply by mifi** [reply, marked as answer]
Did you try one segment from 0-6 and one from 9-10 and then merge?

**Reply by SyedMuhamadYasir** [reply]
ah yes, i figured it out now
 
thanks !

in future iterations, do you think it will be a useful feature to allow us to delete segments from in between as well ? ( like i suggested )

if so, please consider adding it as a feature request to your list

**Reply by mifi** [reply]
There are already two such features. Invert segment mode and invert all segments
